worlds/ff6wc/__init__.py

```diff
@@ -142,6 +142,7 @@ class FF6WCWorld(World):
     def __init__(self, world: MultiWorld, player: int):
         super().__init__(world, player)
         self.starting_characters = None
+        self.starting_espers = None
```

Comment:
This currently crashes for any yaml that doesn't include a flagstring (because this stays `None`).
(It serves as a reminder that it's important to test not only what you intended to change, but also test things you didn't intend to change.)

I don't think we need this to be initialized with `None` - I think it can be initialized with an empty list.